alacritty_terminal/src/term/mod.rs

```diff
@@ -1457,6 +1458,36 @@ impl<T> Term<T> {
             rendered: false,
         }
     }
+
+    fn grid_to_string_between(&self, y0: Line, y1: Line) -> String {
+        let grid = &self.grid;
+        // Use rayon to iterate over each row.
+        (*y0..*y1).into_par_iter().rev().map(|y| {
+            // Create a String and fill it with the contents of the current row.
+            let mut s = String::with_capacity(*grid.cols());
```

Comment:
Allocating a string for every row is probable rather inefficient.

Reply:
Marking this one as resolved because the new version does not do that anymore.